oat-sa/extension-tao-testqti
# Feature/tr 2324/e2e test warnings

Related to [https://oat-sa.atlassian.net/browse/TR-2324](https://oat-sa.atlassian.net/browse/TR-2324)
Cypress test for warning dialogs on navigation

How to run test:
- to setup/run cypress, check description of this PR: https://github.com/oat-sa/extension-tao-testqti/pull/2064
- choose test `views/cypress/tests/delivery/warning-messages-test.spec.js`
- deliveryId is `"warningMessagesTest"` here https://oat-sa.atlassian.net/wiki/spaces/DLV/pages/1539768532/E2E+Testing+on+TAO+delivery+Terre+CG
